specs/clickhouse/clickhouse.spec

```diff
@@ -57,24 +57,21 @@
 
 Summary:           Yandex ClickHouse DBMS
 Name:              clickhouse
-Version:           19.9.4.34
+Version:           19.14.7.15
 Release:           0%{?dist}
 License:           APL 2.0
 Group:             Applications/Databases
 URL:               https://clickhouse.yandex
 
 Source:            %{name}-%{version}.tar.bz2
-Source1:           %{name}.logrotate
```

Comment:
Why did you remove logrotate config from the package?

Reply:
@gongled ?

Reply:
Logrotates is corrupting logs due to ClickHouse rotates its logs itself.

Reply:
Ok, thanks for the explanation.